frontend/src/metabase/public/constants.ts

```diff
@@ -0,0 +1,5 @@
+import { DEFAULT_DASHBOARD_DISPLAY_OPTIONS } from "metabase/dashboard/constants";
+
+export const DEFAULT_EMBED_DISPLAY_OPTIONS = {
```

Comment:
I've tried

```ts
export { DEFAULT_DASHBOARD_DISPLAY_OPTIONS as DEFAULT_EMBED_DISPLAY_OPTIONS } from ....
```

But that confuses the navigating to definition in VS code as it goes to the location of `DEFAULT_DASHBOARD_DISPLAY_OPTIONS` instead of this file, I want to still highlight the fact that this constant is for embedding, but it just has the same value as the one we specified in the dashboard.

I could, however, move the one from dashboard somewhere else e.g. `frontend/src/metabase/resource/constants.ts`, but I don't want to introduce this abstraction yet, I think this is a good compromise for this instance.